migrations/default/1_create_tables/up.sql

```diff
@@ -0,0 +1,157 @@
+drop schema if exists db cascade;
+create schema db;
+
+-------------------------------------------
+-------- lookup tables --------------------
+-------------------------------------------
+create table db.road_types (
+    id integer primary key,
+    description text not null,
+    owner text
+);
+
+create table db.unit_types (
+    id integer primary key,
+    description text not null,
+    owner text not null
+);
+
+insert into db.road_types (id, description, owner) values
+(1, 'alley', 'cris'),
+(2, 'collector', 'cris'),
+(3, 'arterial', 'cris'),
+(4, 'highway', 'cris'),
+(5, 'other', 'cris'),
+(90000, 'urban trail', 'vz');
+
+-- the check constraints prevent us from creating cris-owned values
+-- in our protected vz namespace. we will add additional constraints
+-- to the _cris tables to prevent cris from using our vz lookup values
+alter table db.road_types add constraint road_type_owner_check
+check ((id < 90000 and owner = 'cris') or (id >= 90000 and owner = 'vz'));
+
+insert into db.unit_types (id, description, owner) values
+(1, 'vehicle', 'cris'),
+(2, 'pedestrian', 'cris'),
+(3, 'motorcycle', 'cris'),
+(4, 'truck', 'cris'),
+(5, 'bicycle', 'cris'),
+(6, 'other', 'cris'),
+(90000, 'e-scooter', 'vz');
+
+alter table db.unit_types add constraint unit_type_owner_check
+check ((id < 90000 and owner = 'cris') or (id >= 90000 and owner = 'vz'));
```

Comment:
fwiw, i can't think of scenario where we have to hardcode the custom lookup values in the ETL or the web app; custom lookup values will behave exactly like any other lookup value, and the CRIS import well never reference them.

but, yeah, this definitely approach definitely is a hack. @frankhereford can you help me visualize the alternative you're proposing? i do like the idea of adding our own `id` to our lookup tables. i will for sure add that in the next iteration.

Reply:
John, I was moving too fast trying to get this review put together in advance of our meeting. I had a big confusion here. 

I was conflating using the scale of the ID to denote what is a temporary record, as I've been working with those the last day or two and had them on the mind. I recall now that you're opposed to storing temporary records in the main crashes table, and I'm so fully 100% behind that. 

It makes sense why you mentioned me being interested in how we did lookup table IDs at the end of the meeting now. I do still believe what I was saying earlier that it's always better to let the DB dole out ID numbers and it's very rare to have an occasion where it's best to specify them, like this. 

I think specifying an ID like this is revealing that we're actually storing two different things in one table, which isn't a pattern that I've had success with in the past. In this case, the two things are a CRIS specified lookup value and a VZ specified lookup value. Now, I also agree with you totally that this will work just like any other lookup value, and because the lookup values are generally very stable, it will probably end up being not a big deal. 

But in the interest of doing it as right as we can, I think we should consider a system like I proposed with two tables for the two logical types of input values and a common sequence used to dole out our internal ids so that they can be unioned together easily in a materialized view and be able to use the ID as delivered from either table. 

Beyond advocating for doing it right for the sake of doing it right, having a purely CRIS table will be a huge boon to us if, down the road, CRIS gives us an API where we can pull these values from, because we'd get to skip any logic about > or < 90000 or some arbitrary large number. 

I'm good either way and thank you for asking and catching my oversight and conflation from earlier!